### PR TITLE
@kanaabe => Draft rules for news articles

### PR DESCRIPTION
**Pull request by eessex**
- Adds block map for news layout
- Adds text-nav items for news layout
- Tweaks container width a bit to add padding around news article sections 
- Excludes news articles from 'content-end' dot
- Updates 'NEW_SECTION' action to accommodate articles without sections array

### CODE REVIEW

File: client/apps/edit/components/content/section_controls/index.styl [hunk 1]
```diff
@@ -86,5 +86,5 @@
 [data-layout='news'] .SectionContainer
   &[data-type=image_collection]
     .edit-controls
-      max-width news-body-width
-      margin-left 0px
+      max-width news-body-width-w-margin
+      margin-left -20px
```
Comment (kanaabe) [inline client/apps/edit/components/content/section_controls/index.styl]: Lol thanks, these are better style choices

File: client/apps/edit/components/content/sections/text/index.jsx [hunk 2]
```diff
@@ -73,7 +73,7 @@ export class SectionText extends Component {
 
     // Reset contentEnd markers if end has changed
     if (isContentEnd !== prevProps.isContentEnd) {
-      if (article.layout !== 'classic') {
+      if (['feature', 'standard'].included(article.layout)) {
```
Comment (kanaabe) [inline client/apps/edit/components/content/sections/text/index.jsx]: `included` -> `includes`

Comment (eessex) [reply client/apps/edit/components/content/sections/text/index.jsx]: good catch!

File: client/components/stylus_lib/index.styl [hunk 3]
```diff
@@ -28,6 +28,7 @@ feature-body-w-margin = 720px
 feature-blockquote-width = 940px
 
 news-body-width = 780px
+news-body-width-w-margin = 820px
```
Comment (kanaabe) [inline client/components/stylus_lib/index.styl]: Ohh now I get it -- "with margin" right?

File: client/reducers/editReducer.js [hunk 4]
```diff
@@ -88,7 +88,12 @@ export function editReducer (state = initialState, action) {
       const { section, sectionIndex } = action.payload
       const article = cloneDeep(state.article)
 
-      article.sections.splice(sectionIndex, 0, section)
+      if (article.sections) {
+        article.sections.splice(sectionIndex, 0, section)
+      } else {
+        article.sections = [section]
+      }
```
Comment (kanaabe) [inline client/reducers/editReducer.js]: Cool, does this fix https://artsyproduct.atlassian.net/secure/RapidBoard.jspa?rapidView=4&projectKey=GROWTH&modal=detail&selectedIssue=GROWTH-311&sprint=27 ?

Comment (eessex) [reply client/reducers/editReducer.js]: Yes!

**Comment by kanaabe** [comment]
LGTM! Feel free to merge on 💚

File: client/components/stylus_lib/index.styl (hunk 3 above)
Comment (eessex) [reply client/components/stylus_lib/index.styl]: yeah, is to accommodate the margins for our hover drag/remove controls

**Pull request merged**